### DISCUSSION TITLE
Big issues on Android 13?

### DISCUSSION THREAD

**Discussion by xhatz** [rustdesk/rustdesk#10137] [General]
### Bug Description

Hello! So I switched to a new phone recently, DOOGEE V30Pro, and when I connect to remotely control my laptop, it starts freaking out. When I try to write on the keyboard, the windows just go in fullscreen mode on and off constantly, and some windows even start blinking like going in and out of fullscreen over and over even when I don't click anything.

### How to Reproduce

Connect to my laptop remotely,
Open any window,
Press any key on my phone's keyboard

### Expected Behavior

Just typing characters...

### Operating system(s) on local (controlling) side and remote (controlled) side

Android 13 to Windows 11

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.3.3 (and 1.3.4) to 1.3.3

### Screenshots

.

**Comment by fufesou** [comment]
Maybe it's related to the input method.

RustDesk will reset canvas on metrics changed()What input method did you use? Could you please also give a screenshot to help confirm the issue since 1.3.3(adjust for soft keboard).

What input method did you use? Could you please also give a screenshot to help confirm the issue?

**Reply by xhatz** [reply]
I was using the mouse input but it mostly happened when I used the keyboard I think! I'll try to get a screenshot but it's not really something you can see, it's either making my windows Fullscreen when pressing a keyboard key or making my windows jitter (I think going in and out of Fullscreen over and over)

**Reply by rustdesk** [reply]
How about android 1.3.2？

**Reply by xhatz** [reply]
Same issue sadly with various version...
